pcdsdevices/spectrometer.py

```diff
@@ -339,14 +349,17 @@ class TMOSpectrometer(BaseInterface, GroupDevice, LightpathMixin):
     tab_component_names = True
 
     # Motor components: can read/write positions
-    lens_x = Cpt(BeckhoffAxis, ':MMS:01', kind='normal')
-    lens_pitch_up_down = Cpt(BeckhoffAxis, ':MMS:10', kind='normal')
-    lens_yaw_left_right = Cpt(BeckhoffAxis, ':MMS:11', kind='normal')
-    foil_x = Cpt(BeckhoffAxis, ':MMS:02', kind='normal')
     zone_plate = Cpt(FZPStates, 'SP1K4:FZP:STATE', add_prefix=(), kind='normal')
     zone_plate_x = Cpt(BeckhoffAxis, ':MMS:03', kind='normal')
     zone_plate_y = Cpt(BeckhoffAxis, ':MMS:04', kind='normal')
     zone_plate_z = Cpt(BeckhoffAxis, ':MMS:05', kind='normal')
+    solid_att = Cpt(TMOSpectrometerSOLIDATTStates, 'SP1K4:ATT:STATE', add_prefix=(), kind='normal')
+    # Solid_att x and Y are FOIL x and y
+    solid_att_x = Cpt(BeckhoffAxis, ':MMS:02', doc="X-axis of solid attenuator(FOIL) which protects FZP", kind='normal')
+    solid_att_y = Cpt(BeckhoffAxis, ':MMS:13', doc="Y-axis of solid attenuator(FOIL) which protects FZP", kind='normal')
+    thorlab_lens_x = Cpt(BeckhoffAxis, ':MMS:12', kind='normal')
+    # lens_pitch_up_down = Cpt(BeckhoffAxis, ':MMS:10', kind='normal')
+    # lens_yaw_left_right = Cpt(BeckhoffAxis, ':MMS:11', kind='normal')
```

Comment:
Last small question: is the intention to disable/remove these but be able to bring them back later?

Reply:
yes. James wants to use the two later.